### PR TITLE
fix(staking): add sort numbers

### PR DESCRIPTION
**Pull request by hadzhehsen**

### CODE REVIEW

File: libs/staking/src/lib/components/validator-list-item-desktop.tsx [hunk 1]
```diff
@@ -3,19 +3,24 @@ import type { Validator, DelegationResponse, Reward } from '@evmos/provider';
...
+  sortNumber,
+  sortState,
 }: {
   validator: Validator;
   delegation?: DelegationResponse;
   reward?: Reward;
   stakingPool: number;
   onClick: (validatorAddress: string) => void;
+  sortNumber: number;
+  sortState: SortState;
```
Comment (olegshilov) [inline libs/staking/src/lib/components/validator-list-item-desktop.tsx]:
```suggestion
  index?: number;
```

File: libs/staking/src/lib/components/validator-list-desktop.tsx [hunk 2]
```diff
@@ -198,6 +203,8 @@ export function ValidatorsListDesktop({
               reward={rewardsInfo}
               stakingPool={totalStaked}
               onClick={onValidatorClick}
+              sortNumber={index + 1}
+              sortState={sortState}
```
Comment (olegshilov) [inline libs/staking/src/lib/components/validator-list-desktop.tsx]:
```suggestion
              index={sortState !== 'random' ? index + 1 : undefined}
```

File: libs/staking/src/lib/components/validator-list-desktop.tsx [hunk 3]
```diff
@@ -65,6 +65,11 @@ export function ValidatorsListDesktop({
     <table className="w-full table-fixed">
       <thead className="text-[10px] uppercase leading-[1.2em] text-white/50 md:text-[12px]">
         <tr>
+          {sortState.key !== 'random' && (
+            <th className="w-[40px] p-[8px] text-center lg:p-[12px]">
+              <div className="select-none">#</div>
```
Comment (olegshilov) [inline libs/staking/src/lib/components/validator-list-desktop.tsx]:
```suggestion
              <div className="select-none">&nbsp;</div>
```

**Review by olegshilov — CHANGES_REQUESTED** [review]

File: libs/staking/src/lib/components/validator-list-item-desktop.tsx (hunk 1 above)
Comment (hadzhehsen) [reply libs/staking/src/lib/components/validator-list-item-desktop.tsx]: Done

File: libs/staking/src/lib/components/validator-list-desktop.tsx (hunk 2 above)
Comment (hadzhehsen) [reply libs/staking/src/lib/components/validator-list-desktop.tsx]: Done

File: libs/staking/src/lib/components/validator-list-desktop.tsx (hunk 3 above)
Comment (hadzhehsen) [reply libs/staking/src/lib/components/validator-list-desktop.tsx]: Done

**Review by olegshilov — DISMISSED** [review]

**Review by olegshilov — APPROVED** [review]

**Pull request merged**